Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -122,6 +155,107 @@
                BlueprintName = "XCUITests"
                ReferencedContainer = "container:Client.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
```

Comment:
Why are all these tests disabled?

Reply:
In fennec scheme all XCUITests/UITests are disabled by default. only the ones that will run on nimbledroid are enabled.